packages/react-storage/src/components/StorageBrowser/composables/Title.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import { HeadingElement } from '../context/elements/definitions';
+
+export interface TitleProps {
+  children?: React.ReactNode;
```

Comment:
Afaik, we'll only ever be providing a string to the Title composables. I would suggest that we just do that here too.
```suggestion
  title?: string;
```

Reply:
Got it, for this part I've stuck to the `React.ReactNode` thinking that was a team decision